internal/ydbcp-db-connector/read.go

```diff
@@ -52,36 +52,38 @@ func InitTableClient(dsn string) table.Client {
 		xlog.Error(ctx, "Error connecting to YDB", zap.String("message", err.Error()))
 		return nil
 	}
-	// driver must be closed when done
-	defer func(db *ydb.Driver, ctx context.Context) {
-		err := db.Close(ctx)
-		if err != nil {
-			xlog.Error(ctx, "Error closing YDB driver")
-		}
-	}(db, ctx)
-	return db.Table()
+
+	return db
+}
+
+func (d YdbDriverImpl) Close() {
+	ctx := context.Background()
+	err := d.ydbDriver.Close(ctx)
+	if err != nil {
+		xlog.Error(ctx, "Error closing YDB driver")
+	}
 }
 
 func (d YdbDriverImpl) SelectRunningBackups() chan types.Backup {
 	ctx := context.Background()
-	backups := make(chan types.Backup)
-	err := d.tableClient.Do(ctx, func(ctx context.Context, s table.Session) (err error) {
+	backups := make(chan types.Backup, 100)
+	err := d.ydbDriver.Table().Do(ctx, func(ctx context.Context, s table.Session) (err error) {
 		var (
 			res          result.Result
-			backup_id    uuid.UUID
-			operation_id uint64
+			backup_id    types.ObjectID
+			operation_id *string
 		)
 		_, res, err = s.Execute(
 			ctx,
```

Comment:
Why did you move some queries to .yql files, but not this query?

Reply:
.yql files need to be executed once, this is a different type of query. i'll probably put all of those in a separate .go file as constants later.